src/overlays/actors/ovl_En_Horse_Zelda/z_en_horse_zelda.c

```diff
@@ -177,7 +178,7 @@ void EnHorseZelda_Destroy(Actor* thisx, PlayState* play) {
     Skin_Free(play, &this->skin);
 }
 
-void func_80A6DC7C(EnHorseZelda* this) {
+void EnHorseZelda_ResetAnimation(EnHorseZelda* this) {
```

Comment:
Same as gallop, this looks like a setup for `EnHorseZelda_Stop`

hard to name this action because it still uses the gallop animation despite the actor not moving

Reply:
(I still don't have a better idea for naming the action but at least this should be `_SetupStop` to match `_Stop`)